src/EnergyPlus/CMakeLists.txt

```diff
@@ -19,6 +19,10 @@ if(LINK_WITH_PYTHON)
   add_compile_definitions(LINK_WITH_PYTHON)
 endif()
 
+if(PYTHON_CLI)
+  add_compile_definitions(PYTHON_CLI)
+endif()
```

Comment:
I'd nest it inside the above if(LINK_WITH_PYTHON) to be consistent with the options...

This could be a left over from another pass of CMake, and we shouldn't have a definition "PYTHON_CLI" is "LINK_WITH_PYTHON" is off (I did see you test with `#if LINK_WITH_PYTHON && PYTHON_CLI`, but this could happen if we forget the LINK_WITH_PYTHON)

----

test: 

CMakeLists.txt
```
option(LINK_WITH_PYTHON "Link with Python Library for Python Plugin Builds" OFF)
if(LINK_WITH_PYTHON)
  option(PYTHON_CLI "Build the Auxiliary CLI to Call Python Utilities" OFF)
endif()

if(LINK_WITH_PYTHON)
  message("LINK_WITH_PYTHON")
endif()

if(PYTHON_CLI)
  message("PYTHON_CLI")
endif()
```

```
$ cmake -DLINK_WITH_PYTHON:BOOL=ON -DPYTHON_CLI:BOOL=ON .
LINK_WITH_PYTHON
PYTHON_CLI
 
$ cmake -DLINK_WITH_PYTHON:BOOL=OFF
PYTHON_CLI
```

Reply:
This is fair.  I'll protect the PYTHON_CLI a little better.  As for this being leftover from a cmake pass, are you suggesting other changes need to be made here?  And also - do you think I should do something other than `PYTHON_CLI`?  I thought about `AUXILIARY_CLI` but it seemed that this was a Python-centric thing.  `PYTHON_AUXILIARY_CLI`?  Maybe I just shouldn't worry about the name...

Reply:
Meh, all good choices really.